### PR TITLE
[Term Entry] Python:NumPy Math Methods sin

### LINKED ISSUE #4714: [Term Entry] Python:NumPy Math Methods sin
### Reason/inspiration (optional)

"We would like a new term entry on the `Math Methods` term for [Numpy](https://www.codecademy.com/resources/docs/numpy): sin. The entry should go in a new file under `docs/content/numpy/concepts/math-methods/terms/sin/sin.md`.

The entry should describe the term and how it is used. Also include:
- An "Example" section that explains readers how to use the sin method on a given array.
- Should contain informations for functions and syntax.
- Should have a python codebyte with working code snippet showcasing this function.
Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue."

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by andersooi**
### Description

- Added new term entry for Python:NumPy Math Methods `sin`

### Issue Solved

Closes #4714  

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by letsbuildanything** [comment]
@andersooi, please delete the `.at()` entry file, as you already have a separate PR for it.

File: content/numpy/concepts/math-methods/terms/sin/sin.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+In this codebyte example, the `.sin()` method computes the sine of the elements of the array:
+
+```codebyte/python
+import numpy as np
+
+array = np.array([np.pi/6, np.pi/4, np.pi/2, np.pi])
+result = np.empty_like(array) # create an uninitialised array of same shape
+np.copyto(result, array) # copy original values from array to result
+np.sin(array, where=array >= np.pi/2, out=result)
```
Comment (letsbuildanything) [inline content/numpy/concepts/math-methods/terms/sin/sin.md:76]:
```suggestion
np.sin(array)
```

**Review by letsbuildanything — CHANGES_REQUESTED** [review]
@andersooi Thank you for contributing to the Codecademy Docs!
I've requested some changes. Please take a moment to address them. If you have any concerns or reasons to reject specific suggestions, feel free to discuss them.

**Review by letsbuildanything — APPROVED** [review]
@andersooi Thank you for making the changes.
Ready for the Second round of Review!

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @andersooi, thank you for contributing to docs! 

I've suggested a few changes, could you please review and modify them at your earliest convenience? Thank you! 😄

**Review by mamtawardhani — APPROVED** [review]
Thank you for contributing to docs @andersooi! 😄 

The entry is ready to be merged! 🚀

**Pull request merged**